### PR TITLE
enable periodic test of the topology manager

### PR DESCRIPTION
**Pull request by ffromani**
This PR wants to create periodic jobs for topology manager on branches 4.4 and onwards.
We want to run the topology manager e2e tests against nightlies.

The e2e topology manager tests are a subset of the existing serial conformance tests. They however require a very specific, and likely limited, hardware environment: they require worker nodes with multi-numa hardware with, if possible (not required but strongly recommended) SRIOV VFs.

We create a new  CLUSTER_VARIANT for CLUSTER_TYPE=metal, which configures properly the topology manager (we just need a couple of additional simple manifests), and we create new periodics which run the tests against such CLUSTER_VARIANT.

Please note that we intentionally run _all_ the serial conformance tests, not just the topology manager subset, for extra safety - this environment is seldom tested due to limited availability, so this seems a good compromise.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

**Comment by smarterclayton** [comment]
Oh - you should probably continue using the existing variant I added (multisocket) And populate that loop

File: ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml [hunk 1]
```diff
@@ -408,6 +408,32 @@ objects:
             echo "${CLUSTER_NETWORK_MANIFEST}" > /tmp/artifacts/installer/manifests/cluster-network-03-config.yml
         fi
 
+        if [[ "${CLUSTER_VARIANT}" =~ "numa" ]]; then
```
Comment (smarterclayton) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml:411]: I would start by adding this to the AWS job under multisocket first.  The metal template is out of date and we are in the process of moving it to a new Config, so it’s probably not best place to start.

The other code for multisocket started with this yaml in an earlier state, but note that cluster is a 3 node masters only for cost.

**Comment by fromanirh** [comment]
> Oh - you should probably continue using the existing variant I added (multisocket) And populate that loop

sure I just missed that. Will restart from there.

File: ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml (hunk 1 above)
Comment (fromanirh) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml:411]: ack, thanks.

**Pull request closed without merging**